### PR TITLE
Add monitoring braodcast room to smoketest Matrix setup

### PR DESCRIPTION
**Pull request by ulope**
## Description

Required-for: raiden-network/scenario-player#601

- Add monitoring braodcast room to smoketest Matrix setup
  Needed by the SP smoktest
- Drive-by fix: Add missing log level to json-log-to-html tool

### CODE REVIEW

File: raiden/tests/utils/smoketest.py [hunk 1]
```diff
@@ -306,7 +307,7 @@ def setup_matrix_for_smoketest(
     print_step: Callable,
     free_port_generator: Iterable[Port],
     broadcast_rooms_aliases: Iterable[str],
-) -> Iterator[List[Tuple["ParsedURL", HTTPExecutor]]]:
+) -> Generator[List[Tuple["ParsedURL", HTTPExecutor]], None, None]:
```
Comment (karlb) [inline raiden/tests/utils/smoketest.py:310]: Why the change to Generator?

Comment (ulope) [reply raiden/tests/utils/smoketest.py:310]: Ah that's actually unnecessary. 

I had some "fun" with mypy and at some point it complained that the return type should be `Generator[...]` or one of it's supertypes. But Iterator is a super-type of Generator.

I'll revert that part.

**Review by karlb — APPROVED** [review]
